go/vt/vtadmin/api.go

```diff
@@ -463,6 +469,12 @@ func (api *API) CancelSchemaMigration(ctx context.Context, req *vtadminpb.Cancel
 	span, ctx := trace.NewSpan(ctx, "API.CancelSchemaMigration")
 	defer span.Finish()
 
+	span.Annotate("cluster_id", req.ClusterId)
+
+	if !api.authz.IsAuthorized(ctx, req.ClusterId, rbac.SchemaMigrationResource, rbac.CancelAction) {
+		return nil, fmt.Errorf("%w: cannot cancel schema migration in %s", errors.ErrUnauthorized, req.ClusterId)
```

Comment:
Is the cluster ID part going to be meaningful/helpful to the non-vitess-developer end-user?

Reply:
It might be helpful for users in a multi cluster environment 😄

Reply:
OK :) I just didn't know if the ID is e.g. "2" and if that is easily mapped to a specific cluster for the user in the wider vtadmin context. If so, then all good.

Reply:
Ah! That's good consideration 😄 it will actually be an identifiable cluster name that's passed to the VTAdmin server.